### ISSUE TITLE
JS ERROR: kubesphere-router-kubesphere-system/monitors

### ISSUE THREAD

**Issue by glinkin** [kubesphere/kubesphere#5204]
**Versions Used**
KubeSphere: v3.2.1

**How To Reproduce**
1. /clusters/host/gateways/cluster/kubesphere-router-kubesphere-system/monitors
2. White screen

**Error in console**
```
react_devtools_backend.js:4026 TypeError: Cannot read properties of undefined (reading 'ingress')
    at 1.f9e4cdf45a685a8dc717.js:1:1934301
    at Array.map (<anonymous>)
    at e.getMonitoringCfgs (1.f9e4cdf45a685a8dc717.js:1:1934268)
    at r.value (1.f9e4cdf45a685a8dc717.js:1:1935777)
    at De (main.ea284c62cf0c3cd68d14.js:28:7899)
    at main.ea284c62cf0c3cd68d14.js:28:490778
    at me (main.ea284c62cf0c3cd68d14.js:28:5804)
    at e.track (main.ea284c62cf0c3cd68d14.js:28:17464)
    at r.u [as render] (main.ea284c62cf0c3cd68d14.js:28:490726)
    at Ii (main.ea284c62cf0c3cd68d14.js:86:69876)
```

**Comment by hongzhouzi** [comment]
This issue has been fixed in v3.3, thanks.
PR: https://github.com/kubesphere/console/pull/2900

**Comment by glinkin** [comment]
Thank you. Work 3.3
